### PR TITLE
Add non-AKS cluster setup instructions

### PR DESCRIPTION
**Pull request by markszabo**
As discussed in https://github.com/Azure/kubelogin/issues/69 documentation to describe the setup of non-AKS clusters.

Ref: https://github.com/Azure/kubelogin/issues/51

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -378,6 +378,44 @@ users:
...
+```
+4. Use this credential to connect to the cluster:
+```
+kubectl config set-context "$CLUSTER_NAME" --cluster="$CLUSTER_NAME" --user=azure-user
+kubectl config use-context "$CLUSTER_NAME"
+```
+
+### Known limitation
+
+* [Maximum 200 groups will be included in the OIDC JWT](https://docs.microsoft.com/en-us/azure/active-directory/hybrid/how-to-connect-fed-group-claims)
```
Comment (weinong) [inline README.md:416]: i will also added a note:
for more than 200 groups, consider to use [Application Roles](https://docs.microsoft.com/en-us/azure/active-directory/develop/howto-add-app-roles-in-azure-ad-apps)

**Comment by weinong** [comment]
LGTM. it'd be great if you can include application role reference. its fine if you don't. will merge it next week.

File: README.md [hunk 2]
```diff
@@ -378,6 +378,44 @@ users:
         env: null
 ```
 
+## Non-AKS cluster setup
```
Comment (weinong) [inline README.md:381]: ## Setup for Kubernetes OIDC Provider using Azure AD

Also, please add a link in Features above at L15

**Comment by markszabo** [comment]
Thanks for the comments, I made the changes. Let me know if there is anything else that you'd like me to update.

**Review by weinong — APPROVED** [review]

**Pull request merged**